### PR TITLE
Add the $HOME check for shifuconfig

### PR DESCRIPTION
**Pull request by m4rkl1u**

### CODE REVIEW

**Comment by m4rkl1u** [comment]
Please help to review it

File: src/main/resources/conf/shifuconfig [hunk 1]
```diff
@@ -40,7 +40,7 @@ guagua.zk.embedbed.isInClient=true
 
 # if over such threshold three times, kill worker; by default is 60000, change it to 30 minutes to avoid failures
 # set worker compute time threshold to 30 minutes, if compute time of 3 iterations >= this value, worker will kill itself.
-guagua.computation.time.threshold=1800000
+# guagua.computation.time.threshold=1800000
```
Comment (zhangpengshan) [inline src/main/resources/conf/shifuconfig:43]: Thanks Mark for such PR, could you share why such parameter commented?

Comment (m4rkl1u) [reply src/main/resources/conf/shifuconfig:43]: The configuration is duplicated here https://github.com/ShifuML/shifu/blob/develop/src/main/resources/conf/shifuconfig#L114

**Review by zhangpengshan — APPROVED** [review]
LGTM

**Pull request merged**